app/dashboard/views.py

```diff
@@ -1136,31 +1132,31 @@ def profile(request, handle):
             context['activities'] = paginator.get_page(page)
 
             return TemplateResponse(request, 'profiles/profile_activities.html', context, status=status)
-        else:
-            context = profile.to_dict(tips=False)
-            all_activities = context.get('activities')
-            activity_tabs = []
 
-            for tab, name in tabs:
-                activities = profile_filter_activities(all_activities, tab)
-                activities_count = activities.count()
+        context = profile.to_dict(tips=False)
```

Comment:
Note:  This indent was changed because the above segment returns a response regardless, so if the case is true, it will never make it to else.